charts/dev/README.md

```diff
@@ -0,0 +1,15 @@
+# Tractus-X - Helm Repository: DEV
+Helm charts to deploy Tractus-X products.
```

Comment:
I would add a remark why its DEV; Whats the use case, whats the filter differentiation